front/lib/models/assistant/conversation.ts

```diff
@@ -498,6 +498,9 @@ Message.init(
         unique: true,
         fields: ["conversationId", "rank", "version"],
       },
+      {
+        fields: ["conversationId"],
```

Comment:
Should be covered by the index above?

Reply:
I removed it and instead I updated the query to fetch the reactions to load only the sId of messages, which is the only field we use.